### PR TITLE
[4124][3.1.x] Bulk upload dialog becomes unresponsive with large volume of files

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4124

### CODE REVIEW

File: ui/app/src/components/BulkUpload.tsx [hunk 1]
```diff
@@ -88,6 +95,10 @@ const translations = defineMessages({
     id: 'bulkUpload.uploadInProgress',
     defaultMessage:
       'Uploads are still in progress. Leaving this page would stop the pending uploads. Are you sure you wish to leave?'
+  },
+  maxActiveUploadsReached: {
+    id: 'bulkUpload.maxActiveUploadsReached',
+    defaultMessage: 'Max active uploads reached.'
```
Comment (rart) [inline ui/app/src/components/BulkUpload.tsx]:
```suggestion
    defaultMessage: '{maxFiles} maximum active uploads reached. Excess has been discarded.'
```

Comment (rart) [reply ui/app/src/components/BulkUpload.tsx]: Also update 4

File: ui/app/src/components/BulkUpload.tsx [hunk 2]
```diff
@@ -517,6 +548,11 @@ const DropZone = React.forwardRef((props: DropZoneProps, ref: any) => {
                   </span>
                 )
               })}
+              &nbsp; (
```
Comment (rart) [inline ui/app/src/components/BulkUpload.tsx]: Double check this. Pretty sure this will show the text.
```
              &nbsp; (
```

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/BulkUpload.tsx (hunk 2 above)
Comment (jvega190) [reply ui/app/src/components/BulkUpload.tsx]: It works, but I changed it to be a span with styles for the spacing

File: ui/app/src/components/BulkUpload.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/BulkUpload.tsx]: Updated, the maxFiles parameter in the translation was already being sent (forgot to remove before) so it doesn't need to be added (in 3.1)

**Review by rart — APPROVED** [review]

**Comment by sumerjabri** [comment]
@russdanner do we want this in 3.1.18 or hold off to reduce risk?

**Pull request merged**